pylint_odoo/checkers/no_modules.py

```diff
@@ -327,10 +348,29 @@ class NoModuleChecker(BaseChecker):
         }),
     )
 
+    def open(self):
+        super(NoModuleChecker, self).open()
+        self.config.deprecated_field_parameters = \
+            self.colon_list_to_dict(self.config.deprecated_field_parameters)
+
+    def colon_list_to_dict(self, colon_list=None):
+        """ \
+        Converts a colon list to a dictionary
+        Example:
+            >> self.colon_list_to_dist(['colon:list', 'just_key'])
+            {'colon': 'list', 'just_key': None}
+        """
+        tmp_dict = {}
+        for item in colon_list if isinstance(colon_list, list) else []:
+            prop = item.split(":")
+            tmp_dict[prop[0]] = prop[1] if len(prop) > 1 else None
```

Comment:
What about use `dict([item.split(":") for item in colon_list])`?
![screen shot 2016-12-23 at 3 27 47 pm](https://cloud.githubusercontent.com/assets/6644187/21462630/daa080fc-c923-11e6-974f-f5ed8912b7ea.png)